test/components/Modals/AddContactModal.test.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-import { describe, expect, it, vi } from 'vitest';
+import { describe, it, vi } from 'vitest';
```

Comment:
I'm confused as to why `expect` has been removed. It's a core function from vitest that we're using to validate the unit tests.

Reply:
Hey, I can see how it may look confusing, but expect is still being used in the test. That also answers your other question. except is a global function, and so does not need imported explicitly in each file. the test is still running and passing with except, doing the same thing.

I can change it back if you'd like, it's not a problem.

Reply:
This should be changed back. What the unit tests are using is the `expect` function from `vitest` specifically. It can't be used it if not imported from `vitest` itself. There are no global version of vitest's `expect`